x-pack/test/api_integration/apis/ml/saved_objects/sync.ts

```diff
@@ -89,7 +88,73 @@ export default ({ getService }: FtrProviderContext) => {
       });
     });
 
+    it('should sync datafeeds and saved objects2', async () => {
```

Comment:
Maybe make the test title a bit more connected to the scenario? E.g. something like
```suggestion
    it('should sync datafeeds after recreation in ES with different name', async () => {
```

Reply:
copy and paste mistake